Running: python3 main.py --generations 10 --population 10 --children 10 --selection 1
Running: python3 main.py --generations 10 --population 10 --children 10 --selection 2
Running: python3 main.py --generations 10 --population 10 --children 20 --selection 0
Running: python3 main.py --generations 10 --population 10 --children 20 --selection 1
Running: python3 main.py --generations 10 --population 10 --children 20 --selection 2
Running: python3 main.py --generations 10 --population 50 --children 50 --selection 0
Running: python3 main.py --generations 10 --population 50 --children 50 --selection 1
Running: python3 main.py --generations 10 --population 50 --children 50 --selection 2
Running: python3 main.py --generations 10 --population 50 --children 100 --selection 0
Running: python3 main.py --generations 10 --population 50 --children 100 --selection 1
Running: python3 main.py --generations 10 --population 50 --children 100 --selection 2
Running: python3 main.py --generations 10 --population 10 --children 10 --selection 0 -replacement 1
Running: python3 main.py --generations 10 --population 10 --children 10 --selection 1 -replacement 1
Traceback (most recent call last):
  File "/usr/lib/python3.10/subprocess.py", line 1209, in wait
    return self._wait(timeout=timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1959, in _wait
    (pid, sts) = self._try_wait(0)
  File "/usr/lib/python3.10/subprocess.py", line 1917, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/andrea.ceron/evo/run.py", line 34, in <module>
    subprocess.run(
  File "/usr/lib/python3.10/subprocess.py", line 505, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1146, in communicate
    self.wait()
  File "/usr/lib/python3.10/subprocess.py", line 1222, in wait
    self._wait(timeout=sigint_timeout)
  File "/usr/lib/python3.10/subprocess.py", line 1953, in _wait
    time.sleep(delay)
KeyboardInterrupt
